docker/addons/vault/scripts/vault_set_pki.sh

```diff
@@ -58,9 +77,9 @@ vaultGenerateRootCACertificate() {
         province="\"$MG_VAULT_PKI_CA_ST\"" \
         street_address="\"$MG_VAULT_PKI_CA_ADDR\"" \
         postal_code="\"$MG_VAULT_PKI_CA_PO\"" \
-        ttl=87600h | tee >(jq -r .data.certificate >data/${MG_VAULT_PKI_FILE_NAME}_ca.crt) \
-                         >(jq -r .data.issuing_ca  >data/${MG_VAULT_PKI_FILE_NAME}_issuing_ca.crt) \
-                         >(jq -r .data.private_key >data/${MG_VAULT_PKI_FILE_NAME}_ca.key)
+        ttl=87600h | tee >(jq -r .data.certificate >"$repo_root/docker/addons/vault/scripts/data/${MG_VAULT_PKI_FILE_NAME}_ca.crt") \
```

Comment:
Instead of `$repo_root/docker/addons/vault/scripts/data` use `$scriptdir/data`
And remove `$repo_root`